src/main/java/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package onlinebookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
```

Comment:
it is better to use records for DTO

upgrade your `maven-checkstyle-plugin` to 3.3.0 if Checkstyle fires at records